orderer/consensus/kafka/config.go

```diff
@@ -57,7 +57,7 @@ func newBrokerConfig(
 			Certificates: []tls.Certificate{keyPair},
 			RootCAs:      rootCAs,
 			MinVersion:   tls.VersionTLS12,
-			MaxVersion:   0, // Latest supported TLS version
```

Comment:
Why explicitly set the max version instead of relying on the judgement of the tls package's calculation of max?